doc/man/julia.1

```diff
@@ -216,6 +216,11 @@ expressions. It first tries to use BugReporting.jl installed in current environm
 fallbacks to the latest compatible BugReporting.jl if not. For more information, see
 --bug-report=help.
 
+.TP
+--heap-size-hint=<size>
+Forces garbage collection if memory usage is higher that value. The memory hint might be
+specified in megabytes (500M) or gigabytes (1.5G)
```

Comment:
Why are these in multiples of 500x or 1.5x?

Reply:
those are just examples. It arguably should be written `specified in megabytes (e.g. 500M) or gigabytes (e.g. 1.5G)`

Reply:
> Why are these in multiples of 500x or 1.5x?

`long double` value is used for the numeric part. With 1e-7 restriction as a minimal value. So, even 0.000123G is working.